samples/system-test/v3beta1/translate_create_glossary_beta.test.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2019, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {assert} = require('chai');
+const {TranslationServiceClient} = require('@google-cloud/translate').v3beta1;
+const execa = require('execa');
+const exec = async cmd => (await execa.shell(cmd)).stdout;
+
+const REGION_TAG = 'translate_create_glossary_beta';
+
+describe(REGION_TAG, () => {
+  const translationClient = new TranslationServiceClient();
+
+  it('should create a glossary', async function() {
+    const projectId = await translationClient.getProjectId();
+    const location = 'us-central1';
+    const glossaryId = 'test-glossary';
+    const output = await exec(
+      `node v3beta1/${REGION_TAG}.js ${projectId} ${location} ${glossaryId}`
+    );
+    assert.match(
+      output,
+      /gs:\/\/cloud-samples-data\/translation\/glossary.csv/
+    );
+  }).timeout(10000);
+
+  after('cleanup for glossary create', async function() {
+    this.timeout(10000);
```

Comment:
Instead of adding all these timeouts, can we just bump the timeout in `package.json`?

Reply:
Could you give an example?

e.g. the current [samples/package.json](https://github.com/googleapis/nodejs-translate/blob/master/samples/package.json) doesn't have a timeout configuration

Does Mocha read configuration from package.json?

Reply:
My plan is to add a --timeout 90000 flag (unfortunately it has to be 90000 for the batch translate) to the command in package.json. Lmk if that's not what you were thinking of. Glad to know this is an option! Had no idea before now :)